nbs/Poster.qmd

```diff
@@ -0,0 +1,91 @@
+
+### Challenges:
+
+* Managing large-scale hyperspectral datasets.
+* Balancing reconstruction quality and compression efficiency.
+
+### Contribution:
+
+* Adapted SSCNet from HySpecNet-11k.
+* Applied FasterAI for compressing hyperspectral data.
+* Achieved a practical balance of compression and computational performance.
+
+
+
+## Methodology
+
+### Dataset: HySpecNet-11k
+
+HySpecNet-11k [2] is a large-scale hyperspectral dataset containing 11,483 image patches (128×128 pixels with 224 spectral bands) derived from e Environmental Mapping and Analysis Program (EnMAP) satellite data. It is designed for benchmarking learning-based compression and analysis methods.
+
+* Dataset Splits: Training (70%), Validation (20%), Test (10%).
+* Preprocessing: Removed water vapor-affected bands, applied normalization, and used both patchwise and tilewise splits.
+
+
+### Model: Spectral Signals Compressor Network (SSCNet)
+SSCNet [3] uses 2D convolutions to compress spatial dimensions while preserving spectral integrity.
+
+* Encoder: Three 2D convolutional layers with parametric ReLU activation and max-pooling.
+* Decoder: Uses transposed convolutions for reconstruction.
+* Advantages: Efficient compression with a fixed compression ratio.
+
+### FasterAI Compression:
+
+* Quantization: Reduced bit precision to minimize memory.
+* Pruning: Removed redundant parameters while preserving core model functionality.
+* Integration: FasterAI compresses SSCNet post-training, optimizing it for deployment without sacrificing accuracy.
+
+
+## Experimental Results
+
+
+* SSCNet achieves efficient spectral-spatial compression with competitive peak signal-to-noise ratio (PSNR).
+* FasterAI’s compression significantly reduces model size with minimal loss in PSNR.
+
+
+![](./images/Results.png)
+
+
+### Performance Metrics:
+
+* Compression Ratio: Achieved up to 80% reduction in model size.
+* Accuracy Retention: Maintained high Peak Signal-to-Noise Ratio (PSNR) (~43.64 dB).
+* Inference Efficiency: Substantial reduction in computation time.
+
+### Qualitative Analysis:
+
+* Visuals demonstrate minimal loss of fidelity in reconstructed hyperspectral images.
+
+
+## Conclusion
+
+Our integration of SSCNet with FasterAI compression demonstrates an effective balance of high-quality reconstruction, significant compression, and computational efficiency for hyperspectral imaging tasks.
+
+Impact:
+
+* Enables practical deployment of hyperspectral models in resource-constrained environments.
+* Supports scalable analysis for large datasets like HySpecNet-11k.
+
+## Future Work:
+
+* Test FasterAI compression on additional hyperspectral models.
+* Explore dynamic compression techniques for real-time applications.
```

Comment:
Also you can add: 
- Apply other model compression techniques

This is not clear for me, because I haven't heard about this:
"Explore dynamic compression techniques for real-time applications."
If not related, you can remove it.